src/main/resources/application.properties

```diff
@@ -17,5 +17,5 @@ email.server.domain-host=http://localhost:8080
 email.server.host=smtp.gmail.com
 email.server.port=587
 email.server.username=thehealth02@gmail.com
-email.server.password=wufkpzukvkkmffau
+email.server.password=mmdvbzsqmywrrfby
```

Comment:
Let's extract it into environment variable, like `email.server.password=${EMAIL_SERVER_PASSWORD}`, then you may specify it in run configuration and not store it in the repository